### DISCUSSION TITLE
Group 16 discussion

### DISCUSSION THREAD

**Discussion by jfcarrano** [Digital-Preservation-Coalition/PythonStudyGroups#27] [General]
Discussion about group 16 work

**Comment by emmakhancox** [comment]
Hi everyone, good to see you yesterday. If you notice anything that I missed from our discussion yesterday feel free to add it in or to correct any errors in the Google doc. 😊

https://docs.google.com/document/d/1JWE-ZQxQg6-j7glLtP4F37hjxus8HbrqnwSRqFgn6t4/edit

**Reply by Coffee-Soup** [reply]
Thank you @emmakhancox!

**Comment by Coffee-Soup** [comment]
Here are some possible guides for using Python to manipulate excel files.

- https://automatetheboringstuff.com/2e/chapter13/
- https://realpython.com/openpyxl-excel-spreadsheets-python/
- https://realpython.com/python-csv/

**Comment by jfcarrano** [comment]
I read through the real python csv overview linked above and I have found ["Python for Archivists: Breaking Down Barriers Between Systems"](https://hdl.handle.net/1813/76861) a useful overview in the past as well.

**Reply by Coffee-Soup** [reply]
@jfcarrano thank you for sharing, this looks great.
